code/statsd/src/main/scala/com/avast/datadog4s/statsd/StatsDEventFactory.scala

```diff
@@ -0,0 +1,21 @@
+package com.avast.datadog4s.statsd
+
+import cats.effect.Sync
+import com.avast.datadog4s.StatsDMetricFactoryConfig
+import com.avast.datadog4s.api.event.Eventer
+import com.avast.datadog4s.api.{ EventFactory, Tag }
+import com.avast.datadog4s.statsd.event.EventerImpl
+import com.timgroup.statsd.StatsDClient
+
+class StatsDEventFactory[F[_]: Sync](statsDClient: StatsDClient, basePrefix: String, config: StatsDMetricFactoryConfig)
+    extends EventFactory[F] {
+  import config.defaultTags
```

Comment:
i think its preferable to just use `config.defaultTags` in the code without having this import here, it's make it more obvious where the value is coming from

Reply:
good catch